cluster/images/etcd/migrate/migrate_server.go

```diff
@@ -108,7 +108,7 @@ func (r *EtcdMigrateServer) Stop() error {
 	timedout := make(chan bool)
 	go func() {
 		time.Sleep(gracefulWait)
-		timedout <- true
+		close(timedout)
```

Comment:
Given you're touching it, can you please also fix line 72:
https://github.com/kubernetes/kubernetes/blob/96360055842db86ff883993824a9b9bd0563b58e/cluster/images/etcd/migrate/migrate_server.go#L72

?

Reply:
done